README.md

```diff
@@ -254,42 +254,26 @@ provides(Sources,URI("http://libvirt.org/sources/libvirt-1.1.1-rc2.tar.gz"),libv
 
 # The high level interface - Loading dependencies
 
-BinDeps provides the `@BinDeps.load_dependencies` macro that you may call early in 
-initialization process of your package to load all declared libraries in your build.jl
-file. 
-
-The basic usage is very simple:
-
-```jl
-using BinDeps
-@BinDeps.load_dependencies
+To load dependencies without a runtime dependence on BinDeps, place code like the following
+near the start of the Package's primary file. Don't forget to change the error message to 
+reflect the name of the package.
+```julia
+const depsfile = joinpath(dirname(@__FILE__), "..", "deps", "deps.jl")
```

Comment:
I see now. But isn't it best practice to encourage the latest syntax and let Compat take care of backwards compatibility.